### PR TITLE
API endpoint for access token

### PR DESCRIPTION
**Pull request by micahelinor**
Routes to announcement page if the access token is correct and is in local storage

### CODE REVIEW

File: backend/withAccess.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require("dotenv").config;
+const accessToken = process.env.accessToken;
+
+const checkAccess = function(req, res) {
+  const token = req.headers.authorization;
+  console.log(token);
```
Comment (trev4ev) [inline backend/withAccess.js]: no console.log please!

File: src/components/Login.js [hunk 2]
```diff
@@ -16,6 +16,32 @@ class Login extends Component {
...
+    const response = await fetch("http://localhost:3001/checkToken", {
+      headers: {
+        authorization: localStorage.get("token"),
+        Accept: "application.json"
+      }
+    });
+    if (response.status === 401) {
+      let path = "/login";
+      this.props.history.push(path);
+    }
```
Comment (trev4ev) [inline src/components/Login.js]: why is this part necessary, I don't think we ever send a 401 error and we would just be redirecting to the same route

File: src/components/Login.js [hunk 3]
```diff
@@ -16,6 +16,32 @@ class Login extends Component {
...
+    const json = await response.json();
+    if (
+      json.result === true &&
+      this.props.history.location.pathname === "/login"
```
Comment (trev4ev) [inline src/components/Login.js]: why do have to make this check? I think it should always be true

File: backend/app.js [hunk 4]
```diff
@@ -13,33 +13,33 @@ const userId = process.env.userId;
...
 const readline = require("readline");
-const {google} = require("googleapis");
+const { google } = require("googleapis");
 var fs = require("fs");
 const ordersdb = require("./orderquery");
 const bcryptjs = require("bcryptjs");
 app.use(cors());
 app.use(bodyParser.json());
 const withAuth = require("./middleware");
+const checkAccess = require("./withAccess");
```
Comment (trev4ev) [inline backend/app.js]: the naming on this is a little weird, can you rename the file to checkAccess as well?

**Review by trev4ev — CHANGES_REQUESTED** [review]
Please make these small changes!

**Review by abaruah117 — APPROVED** [review]
lgtm
